javascript/core/errors/exceptions.md

```diff
@@ -32,9 +32,9 @@ links:
 
 
 aspects:
-  - introduction
+  - deep
```

Comment:
I would revert these changes, it's certainly an `introduction` level insight, and not `obscura`.